File: abc/abc206/a.cs
// https://atcoder.jp/contests/abc206/submissions/46502984

using System;

public class A
{
  static void Main()
  {
    int n = int.Parse(Console.ReadLine());
    int s = n * 108 / 100;

    if(s < 206)
    {
      Console.WriteLine("Yay!");
    }
    else if(s > 206)
    {
      Console.WriteLine(":(");
    }
    else
    {
      Console.WriteLine("so-so");
    }
  }
}

File: abc/abc206/a.rs
// https://atcoder.jp/contests/abc206/submissions/47525335

use proconio::input;

fn main() {
    input! {
        n: u16
    }

    let x = n * 108 / 100;
    print!("{}", if x < 206 {"Yay!"} else if x > 206 {":("} else {"so-so"});
}

